licenses/LICENSE

```diff
@@ -0,0 +1,550 @@
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+
+=======================================================================
+Apache servicecomb-kie Subcomponents:
+
+The Apache servicecomb-kie project contains subcomponents with
+seperate copyright notices and license terms. Your use of the binary release
+for these subcomponents is subject to the terms and conditions of the
+following licenses.
+
+
+================================================================
+The following component(s) are provided under the Apache-2.0 License (Apache License 2.0).
+See the respective project link for details.
+
+github.com/modern-go/concurrent (v0.0.0-20180306012644-bacd9c7ef1dd)
+github.com/modern-go/reflect2 (v1.0.1)
+github.com/prometheus/client_golang (v0.9.1)
+github.com/prometheus/client_model (v0.0.0-20190115171406-56726106282f)
+github.com/prometheus/common (v0.2.0)
+github.com/prometheus/procfs (v0.0.0-20190117184657-bf6a532e95b1)
+go.mongodb.org/mongo-driver (v1.0.3)
+github.com/matttproud/golang_protobuf_extensions (v1.0.1)
+github.com/mwitkow/go-conntrack (v0.0.0-20161129095857-cc309e4a2223)
+gopkg.in/yaml.v2 (v2.2.1)
+github.com/xdg/scram (v0.0.0-20180814205039-7eeb5667e42c)
+github.com/xdg/stringprep (v1.0.0)
+github.com/go-mesh/openlogging (v1.0.1-0.20181205082104-3d418c478b2d)
+
+================================================================
+The following component(s) are provided under the Apache-2.0 License (Apache License 2.0).
+See the respective project link for details.
+You can find a copy of the License at LICENSE_Apache2.0 .
```

Comment:
It's go-chassis related license, we should keep the name of  go-chassis license.

Reply:
go-chassis license is apache-2.0. I change the file name from  LICENSE_go-chassis_go-chassis to LICENSE_Apache2.0 as you told me before. Should I change it back?